files/haproxy.cfg

```diff
@@ -37,6 +38,12 @@ defaults
   # Without the knowledge that our client is ELB this value doesn't make sense
   timeout client 3600s
   timeout server 3600s
+{{- else }}
+  timeout http-keep-alive 60s
```

Comment:
Good call. I'll add to another PR since we both have to approve again if more commits are pushed